### ISSUE TITLE
Progress out of Fab

### ISSUE THREAD

**Issue by AGr7K** [JorgeCastilloPrz/FABProgressCircle#3]
in android 4.3 and android 4.4.2 same issue
![device-2015-06-09-174105](https://cloud.githubusercontent.com/assets/3106973/8060644/3bfbf5fe-0ecf-11e5-8ec3-48adb1374c76.png)
![device-2015-06-09-174127](https://cloud.githubusercontent.com/assets/3106973/8060645/3bfeeda4-0ecf-11e5-9e2e-e04f5c67ce38.png)

thank you

**Comment by JorgeCastilloPrz** [comment]
:o ! Thank you very much for reporting. I will work in a fix asap.

**Comment by JorgeCastilloPrz** [comment]
In the meantime, it will be very nice if you can paste hre a copy of your XML declaration. That might help me a little bit.

**Comment by AGr7K** [comment]
in android < 5.0 
app:elevation="0dp" but its same issue

<com.github.jorgecastilloprz.FABProgressCircle
        android:id="@+id/fabProgressCircle"
        android:layout_width="wrap_content"
        android:layout_margin="16dp"
        app:arcColor="?colorPrimary"
        app:arcWidth="7dp"
        android:layout_height="wrap_content">

        <android.support.design.widget.FloatingActionButton
            android:id="@+id/pick"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:clickable="true"
            app:borderWidth="0dp"
            app:elevation="0dp"
            android:src="@drawable/abc_ic_ab_back_mtrl_am_alpha"
            app:layout_anchor="@id/toolbar_top"
            />
    </com.github.jorgecastilloprz.FABProgressCircle>

**Comment by JorgeCastilloPrz** [comment]
This issue and all issues related to this one have been fixed in patch 1.01. You already can download it.
```groovy
dependencies{
    compile 'com.github.jorgecastilloprz:fabprogresscircle:1.01@aar'
}
```
or 
```xml
<dependency>
    <groupId>com.github.jorgecastilloprz</groupId>
    <artifactId>fabprogresscircle</artifactId>
    <version>1.01</version>
    <type>aar</type>
</dependency>
```
